src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final int maxNumber = 101;

```

Comment:
it should be static final constant 

[Don't forget how to name constants according to style guide (the same with naming of enum values)](https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#dont-forget-how-to-name-constants-according-to-style-guide-the-same-with-naming-of-enum-values)

Reply:
fix comment